XamlControlsGallery/ControlPages/InfoBarPage.xaml

```diff
@@ -0,0 +1,122 @@
+                </StackPanel>
+            </local:ControlExample.Options>
+
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;InfoBar 
+    IsOpen="$(IsOpen)"
+    Severity="$(Severity)"
+    Title="Title"
+    Message="Essential app message for your users to be informed of, acknowledge, or take action on." /&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Severity" Value="{x:Bind SeverityComboBox.SelectedValue, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="IsOpen" Value="{ x:Bind IsOpenCheckBox1.IsChecked.Value, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+        <local:ControlExample x:Name="Example2" HeaderText="A closable InfoBar with a long or short message and various buttons">
+            <local:ControlExample.Example>
+                <controls:InfoBar x:Name="TestInfoBar2" AutomationProperties.Name="TestInfoBar2"
+                IsOpen="True"
+                Title="Title" />
+            </local:ControlExample.Example>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="IsOpen" Value="{ x:Bind IsOpenCheckBox2.IsChecked.Value, Mode=OneWay}" />
+                <local:ControlExampleSubstitution x:Name="DisplayMessage" Key="DisplayMessage" Value=""/>
+                <local:ControlExampleSubstitution x:Name="DisplayButton" Key="DisplayButton" Value=""/>
+            </local:ControlExample.Substitutions>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <CheckBox x:Name="IsOpenCheckBox2" AutomationProperties.Name="IsOpenCheckBox2" Content="Is Open" IsChecked="{x:Bind TestInfoBar2.IsOpen, Mode=TwoWay}"/>
+                    <ComboBox x:Name="MessageComboBox" AutomationProperties.Name="MessageComboBox" Header="Message Length" SelectionChanged="MessageComboBox_SelectionChanged" SelectedIndex="1">
+                        <ComboBoxItem Content="Short" />
+                        <ComboBoxItem Content="Long" />
+                    </ComboBox>
+                    <ComboBox x:Name="ActionButtonComboBox" AutomationProperties.Name="ActionButtonComboBox" Header="Action Button" SelectionChanged="ActionButtonComboBox_SelectionChanged" SelectedIndex="0">
+                        <ComboBoxItem Content="None" />
+                        <ComboBoxItem Content="Button" />
+                        <ComboBoxItem Content="Hyperlink" />
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;InfoBar 
+    IsOpen="$(IsOpen)"
+    Title="Title"
+    Message="$(DisplayMessage)"
+    $(DisplayButton)/&gt;
+                </x:String>
```

Comment:
This can result in faulty XAML when switching to button or hyperlink "action" content:
![Image of faulty XAML sample code for action button state](https://user-images.githubusercontent.com/16122379/99296974-dff38680-2847-11eb-83b0-36f79c972f38.png)

Reply:
I've fixed this by changing the bracket format, however when the button is not applied, there's an extra space.
![image](https://user-images.githubusercontent.com/20007011/99785112-928a4a00-2ad1-11eb-9f9f-26b1bcf60d3d.png)

Reply:
Looking good! I'm not too worried about the extra whitespace, we can fix that later if needed.